webapp/src/components/modals/actionModal.tsx

```diff
@@ -0,0 +1,171 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Table from 'react-bootstrap/Table';
+import Form from 'react-bootstrap/Form';
+import {OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+import './styles.scss';
+
+type Props = {
+    visibility: boolean;
+    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
+    config: Config[];
+    configIndex: number;
+}
+
+const ActionModal = ({visibility, setVisibility, config, configIndex}: Props) => {
+    const [isAttachmentMessageAvailable, setIsAttachmentMessageAvailable] = useState(false);
+
+    const actionsLength = config[configIndex]?.actions?.length;
+
+    const checkAttachmentMessage = () => {
+        if (config[configIndex]?.attachmentMessage?.length) {
+            setIsAttachmentMessageAvailable(Boolean(config[configIndex]?.attachmentMessage?.[0]));
+            return;
+        }
+
+        setIsAttachmentMessageAvailable(false);
+    };
+
+    useEffect(() => {
+        checkAttachmentMessage();
+    }, []);
+
+    const handleClose = () => setVisibility(false);
+
+    return (
+        <Modal
+            className='action-modal'
+            show={visibility}
+            onHide={handleClose}
+        >
+            <Modal.Header closeButton={false}>
+                <Modal.Title>{'Actions'}</Modal.Title>
+            </Modal.Header>
+
+            <Modal.Body className='action-modal__body'>
+                {isAttachmentMessageAvailable || (config[configIndex]?.actions && actionsLength) ? (<>
```

Comment:
```suggestion
                {isAttachmentMessageAvailable || actionsLength ? (<>
```
I dont think this is needed